infra/terraform/azure/README.md

```diff
@@ -0,0 +1,36 @@
+# Terraform config for Feast on Azure
+
+This serves as a guide on how to deploy Feast on Azure. At the end of this guide, we will have provisioned:
+1. AKS cluster
+2. Feast services running on AKS
+3. Azure Cache (Redis) as online store
+4. Spark operator on AKS
+5. Kafka running on HDInsight.
```

Comment:
By the way, is there a reason we are using HDInsights over Event Hubs for Kafka?

Reply:
HDInsight actually runs Kafka (it's essentially Kafka managed by Azure), whereas Event Hubs is Kafka-client-compatible but doesn't support certain Kafka features. Seemed faster and more likely to work using HDInsight